src/tools/pub/cmd/pub-upgrade.md

```diff
@@ -1,56 +1,56 @@
 ---
-title: pub upgrade
-description: Use pub upgrade to get the latest versions of all dependencies used by your Dart app.
+title: dart pub upgrade
+description: Use dart pub upgrade to get the latest versions of all dependencies used by your Dart app.
 ---
 
 _Upgrade_ is one of the commands of the [pub tool](/tools/pub/cmd).
```

Comment:
We should probably say something about `pub upgrade` vs. `dart pub upgrade`, using some standard wording. (Also see #2826.)